Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1802,7 +1802,7 @@ def lhegensim(fragment,howMuch):
 
     if cust!=None : upgradeStepDict['DigiFull'][k]['--customise']=cust
     if era is not None: upgradeStepDict['DigiFull'][k]['--era']=era
- 
+    
```

Comment:
please avoid making (what seem like totally pointless) whitespace-only changes to files